code/datums/material_container.dm

```diff
@@ -143,13 +143,12 @@
 		if(M.amount < (sheet_amt * MINERAL_MATERIAL_AMOUNT))
 			sheet_amt = round(M.amount / MINERAL_MATERIAL_AMOUNT)
 		var/count = 0
-
 		while(sheet_amt > MAX_STACK_SIZE)
 			new M.sheet_type(get_turf(owner), MAX_STACK_SIZE)
 			count += MAX_STACK_SIZE
 			use_amount_type(sheet_amt * MINERAL_MATERIAL_AMOUNT, M.id)
 			sheet_amt -= MAX_STACK_SIZE
-
+			M.amount = (sheet_amt * MINERAL_MATERIAL_AMOUNT)
```

Comment:
That is what I am referring to, yes.

My query arose due to ```use_amount_type()``` using the same calculation you're using for the sheet's new amount, except since ```sheet_amt``` is modified before you repeat the calculation, these two numbers aren't actually the same (when it *reads* as if they should be)

Otherwise the amount of material in the sheet is less than the amount used, which is weird.